src/classLibrary/Parser.ts

```diff
@@ -0,0 +1,146 @@
+import { IParsingStrategy, 
+    VirtualFileSystemInstance,
+    CyanSafe
+} from "./interfaces/interfaces";
+import chalk from "chalk";
+import { GuidResolver } from "./ParsingStrategies/GuidResolver";
+import { IfElseResolver } from "./ParsingStrategies/IfElseResolver";
+import { InlineFlagResolver } from "./ParsingStrategies/InlineFlagResolver";
+import { VariableResolver } from "./ParsingStrategies/VariableResolver";
+import { Utility } from "./Utility/Utility";
+import {Bar, Presets} from "cli-progress";
```

Comment:
If possible, change this to follow the previous import styles

Reply:
What is the previous import styles?

Reply:
you mean the space? LOL